### ISSUE TITLE
[Other - Accessibility] Redundant link text announced by screen readers. (00.00.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#52250]
### Discussed in https://github.com/department-of-veterans-affairs/va.gov-team/discussions/52244

<div type='discussions-op-text'>

<sup>Originally posted by **shiragoodman** November 29, 2022</sup>
#### VFS team name
Sitewide Content

#### VFS product name
Education rates pages

#### VFS feature name
Education rates pages

#### Point of Contact/Reviewers
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
No VA.gov Experience Standard for the issue found.

### Issue Details
Several links on both pages have `title` text that doesn't seem to add any value. The `title` attribute should be used on links only when the extra text in the `title` attribute clarifies the purpose of the link for screen reader users -- and usually, the better option is just writing unambiguous visible link text and not worry about the `title` attribute at all. On these pages, I'm mainly seeing redundant `title` text, which is a bit of a headache for screen reader users.

### Link, screenshot or steps to recreate
Two examples:

On the Post-9/11 GI Bill Rates page, under "Full rates for school and training programs" > "Added payments," we have:
```
<a ... href="/education/about-gi-bill-benefits/post-9-11/yellow-ribbon-program" title="Yellow Ribbon Program">
   Learn about the Yellow Ribbon Program
</a>
``` 
Many screen readers will announce both the visible text and the title text. For example, NVDA announces this as `Learn about the Yellow Ribbon Program, link, Yellow Ribbon Program`.

On the Current VA Education Benefit Rates page, under "Current VA education benefit rates by program" > "Post-9/11 GI Bill (Chapter 33) rates," we have:

```
<a ... href="/post-911-gi-bill-chapter-33-rates" title="Post-9/11 GI Bill (Chapter 33) rates" ...>
   Check the current rates for the Post-9/11 GI Bill
</a>
```

NVDA announces this as `Check the current rates for the Post-9/11 GI Bill, link, Post-9/11 GI Bill (Chapter 33) rates`.

### VA.gov Experience Standard
[Category Number 00, Issue Number 00](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Platform Recommendation
Remove `title` attributes from links when they don't clarify the purpose of the link.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.</div>

**Comment by shiragoodman** [comment]
I accidentally converted this issues to a discussion, and then back to an issue. I was just trying to remove the `sitewide-content` label and add the `sitewide content` label

**Comment by megzehn** [comment]
Thanks, @shiragoodman!

@laflannery, could you weigh in to assess if there are still unnecessary title attributes in the nodes and links referenced here? I'm not seeing them in the source code. If they're still there, could you confirm if we have the ability to remove title attributes in Drupal? 

Nodes and links that I reviewed based on this ticket:

[Past rates for Post-9/11 GI Bill](https://prod.cms.va.gov/node/49728/latest)
Links reviewed:
- [Learn about the Yellow Ribbon Program](http://preview-prod.vfs.va.gov/education/about-gi-bill-benefits/post-9-11/yellow-ribbon-program) (Link under the H2 **Full rates for school and training programs**, and the accordion "Added payments.")
- [Learn what tests we approve for the GI Bill](https://prod.cms.va.gov/education/about-gi-bill-benefits/how-to-use-benefits/test-fees) (Link under H2 **Full rates for testing fees**. I did see a title attribute for this link and did attempt to remove it.)

[Post-9/11 GI Bill (Chapter 33) rates ](https://prod.cms.va.gov/node/1009/latest)
Links reviewed:
- [Learn about the Yellow Ribbon Program](http://preview-prod.vfs.va.gov/education/about-gi-bill-benefits/post-9-11/yellow-ribbon-program) (Link under the H2 **Full rates for school and training programs**, and the accordion "Added payments.")
- [Learn what tests we approve for the GI Bill](https://prod.cms.va.gov/education/about-gi-bill-benefits/how-to-use-benefits/test-fees) (Link under H2 **Full rates for testing fees**. I did see a title attribute for this link and did attempt to remove it.)

[Current education benefit rates](https://prod.cms.va.gov/node/938/revisions)
Link reviewed:
- [Check the current rates for the Post-9/11 GI Bill](http://preview-prod.vfs.va.gov/education/benefit-rates/post-9-11-gi-bill-rates)(Link under the H3 **Post-9/11 GI Bill (Chapter 33) rates**)

**Comment by laflannery** [comment]
@megzehn There is a [Drupal issue](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/11843) where title attributes are automatically added when you link to another node, unfortunately this is not something you can control - the CMS team needs to resolve this.  We do have this on our radar though.

That being said - I'm not sure what you would do with this ticket? Do you have a "blocked" status/label? Or would you close it because you have done all you can and this is now on the CMS team and tracked within the linked ticket?

**Comment by megzehn** [comment]
Thanks for confirming, @laflannery! Yes, we can add this to blocked for now. CC @RLHecht!
